src/main/resources/assets/mmorpg/lang/zh_cn.json

```diff
@@ -1,113 +1,98 @@
+	"block.mmorpg.ore_block5": "玄品矿石",
+	"itemGroup.mmorpg_main": "挖矿与砍杀",
+	"itemGroup.mmorpg_uniques": "珍品装备",
+	"itemGroup.mmorpg_currency": "合成物品",
+	"itemGroup.mmorpg_lootboxes": "战利品盒",
+	"itemGroup.mmorpg_alchemy": "炼药",
+	"block.mmorpg.modify_station": "装备改造台",
+	"block.mmorpg.repair_station": "装备修理台",
+	"block.mmorpg.salvage_station": "装备回收台",
+	"block.mmorpg.map_device": "地图装置",
+	"block.mmorpg.alchemy_station": "炼药台",
+	"mmorpg.tooltip.item_modifiable_in_station": "可以在装备改造台中使用",
+	"mmorpg.tooltip.lootbox": "盒子里装着些什么呢？",
+	"mmorpg.tooltip.player_levelup": "如果你的砍杀经验条已满，右击此物品便可突破等级！",
+	"mmorpg.tooltip.newbie_gear_bag": "开启后可获得一整套开局装备。",
+	"mmorpg.lore.newbie_gear_bag1": "三个熔炉底一行，五个铁锭半圈放，",
+	"mmorpg.lore.newbie_gear_bag2": "正中置以小种子，种下我们的希望！",
+	"mmorpg.tooltip.capacitor": "放在装备修理台中以减少燃料的消耗。",
+	"mmorpg.tooltip.capacitor2": "燃料消耗比率",
+	"mmorpg.tooltip.socketable_item": "这块宝石可以插入插槽",
+	"item.mmorpg.player_levelup": "玩家升级令牌",
+	"item.mmorpg.loot_bag": "装备战利品袋",
+	"item.mmorpg.map_bag": "地图袋",
+	"item.mmorpg.currency_bag": "通货袋",
+	"item.mmorpg.newbie_gear_bag": "新手装备袋",
+	"item.mmorpg.tutorial_book": "挖矿与砍杀教程",
+	"item.mmorpg.map_back_portal": "地图回返珍珠",
+	"item.mmorpg.level_nearest_entity": "提升邻近实体的等级",
+	"item.mmorpg.increase_rarity_nearest_entity": "提升邻近实体的品级",
+	"item.mmorpg.mat.infused_iron": "注源铁锭",
+	"item.mmorpg.mat.crystallized_essence": "晶化源质",
+	"item.mmorpg.mat.golden_orb": "灿金宝珠",
+	"item.mmorpg.mat.mythic_essence": "神话源质",
+	"item.mmorpg.awaken_runeword": "觉醒符文之语",
+	"item.mmorpg.identify_tome": "鉴定卷轴",
+	"item.mmorpg.identify_tome_plus": "鉴定卷册",
+	"item.mmorpg.teleport_scroll": "返回卷轴",
+	"item.mmorpg.prefix": "前缀",
+	"item.mmorpg.suffix": "后缀",
+	"block.mmorpg.scrabble": "谜语箱",
+	"block.mmorpg.map_chest": "地图箱子",
+	"mmorpg.tooltip.aoe_spell_explosion": "释放一枚元素弹射物，命中后造成范围伤害。",
+	"mmorpg.tooltip.single_target_spell_bolt": "释放一枚元素弹射物，对单个目标造成伤害。",
+	"mmorpg.tooltip.spell_instant_heal": "立即治疗施法者。",
+	"mmorpg.tooltip.aoe_bomb_spell": "留下一枚炸弹，之后造成范围爆炸伤害。",
+	"mmorpg.tooltip.spell_self_regen": "持续治疗施法者的生命值。",
+	"mmorpg.tooltip.aoe_spell_nova": "伤害你周围的所有目标。",
+	"mmorpg.tooltip.ele_bonus_atk_dmg_spell": "给予增益效果，消耗法力造成额外伤害。",
+	"item.mmorpg.capacitor0": "凡品能效珠",
+	"item.mmorpg.capacitor1": "下品能效珠",
+	"item.mmorpg.capacitor2": "中品能效珠",
+	"item.mmorpg.capacitor3": "上品能效珠",
+	"item.mmorpg.capacitor4": "精品能效珠",
+	"item.mmorpg.capacitor5": "玄品能效珠",
+	"death.attack.mmorpg.custom_damage": "%1$s被残暴地杀害了",
+	"mmorpg.name.map": "冒险地图",
+	"mmorpg.tooltip.put_in_mapdevice": "右击地图装置使用",
+	"mmorpg.tooltip.press_shift_more_info": "[Shift]：信息，[Alt]：描述，[Ctrl]：附魔",
+	"mmorpg.tooltip.affix_rarity_lootbonus": "战利品词缀品级增益",
+	"curios.identifier.salvage_bag": "回收袋",
+	"curios.identifier.bracelet": "手镯",
+	"entity.mmorpg.trader.none": "地牢商人",
+	"block.mmorpg.dungeon_portal": "地牢传送门",
+	"entity.mmorpg.spirit_wolf_pet": "召唤猎狼",
+	"entity.mmorpg.zombie_pet": "召唤僵尸",
+	"entity.mmorpg.skeleton_pet": "召唤骷髅",
+	"entity.mmorpg.archon_pet": "召唤执政官",
```

Comment:
I just wonder the "archon" is the same "archon" in the StarCraftⅡ? Anymay I used the same translation of it. If it's not please let me know.